web/template/timeline/deputy-details-contact-changed.gotmpl

```diff
@@ -20,3 +20,4 @@
 {{ end }}
 
 
+
```

Comment:
how many empty lines does this file need 😄

Reply:
Josh, this comment made me chuckle!